src/lib/createElement.js

```diff
@@ -1,5 +1,50 @@
+import { InvalidVNodeTypeError } from "../errors/InvalidVNodeTypeError";
+import { isRenderableVNode, isTextNode } from "../utils/domUtils";
+import { getEventTypeFromProps } from "../utils/eventUtils";
 import { addEvent } from "./eventManager";
 
-export function createElement(vNode) {}
+export function createElement(vNode) {
+  if (typeof vNode === "function") {
+    throw new InvalidVNodeTypeError();
+  }
 
-function updateAttributes($el, props) {}
+  if (!isRenderableVNode(vNode)) {
+    return document.createTextNode("");
+  }
+
+  if (isTextNode(vNode)) {
+    return document.createTextNode(vNode);
+  }
+
+  if (Array.isArray(vNode)) {
+    const docFragment = document.createDocumentFragment();
+    const childNodes = vNode.map(createElement);
+    childNodes.forEach((childNode) => docFragment.appendChild(childNode));
+    return docFragment;
+  }
+
+  const $element = document.createElement(vNode.type);
+  for (let [key, value] of Object.entries(vNode.props || {})) {
+    addAttirbutes($element, key, value);
+  }
+  const childNodes = (vNode.children || []).map(createElement);
+  childNodes.forEach((childNode) => $element.appendChild(childNode));
+  return $element;
+}
+
+export const addAttirbutes = (element, key, value) => {
```

Comment:
오타인 것 같은데 `addAttributes`로 수정하면 좋을 것 같습니다~
```suggestion
export const addAttributes = (element, key, value) => {
```